### PR TITLE
Adds onboarding documentation

### PR DESCRIPTION
**Pull request by mcardosos**

### CODE REVIEW

**Comment by mcardosos** [comment]
@troydai :D

File: onboard.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    }
+  },
+  {
+    "ver": "1.0",
+    "execution": {
+      "command": "go test -v github.com\\Azure-Samples\\azure-sdk-for-go-samples\\storage -run ^ExampleUploadBlockBlob$"
+    },
+    "classifier": {
+      "identifier": "storage/UploadBlockBlob",
+      "type": "Live"
```
Comment (troydai) [inline onboard.md]: The `type` is not required.

File: onboard.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    }
+  }
+]
+```
+
+This example is showing all the required fields. However, the test list might also contain other fields.
+
+- Have an `app` folder with:
+  - `get_index` program. This program should print the test list.
+  - `a01droid` binary, you can get it in https://a01tools.blob.core.windows.net/droid/linux/a01droid Find more [info about the a01droid](https://github.com/Azure/adx-automation-agent).
```
Comment (troydai) [inline onboard.md]: Path: `https://a01tools.blob.core.windows.net/droid/latest/linux/a01droid`

**Review by troydai — APPROVED** [review]
Just update these two places you're good to go.

File: onboard.md (hunk 1 above)
Comment (mcardosos) [reply onboard.md]: Fixed

File: onboard.md (hunk 2 above)
Comment (mcardosos) [reply onboard.md]: Fixed

**Comment by troydai** [comment]
Done. Thanks.

**Pull request merged**